core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -3008,6 +3008,12 @@ private void checkLiteral2(String expression, String expected) {
             + "OVER (PARTITION BY \"hire_date\" ORDER BY \"employee_id\"), \"hire_date\"\n"
             + "FROM \"foodmart\".\"employee\"\n"
             + "GROUP BY \"hire_date\", \"employee_id\"";
+    String query7 = "SELECT "
+        + "count(distinct \"employee_id\") over (order by \"hire_date\") FROM \"employee\"";
+    String expected7 = "SELECT "
+        + "COUNT(DISTINCT \"employee_id\") "
+        + "OVER (ORDER BY \"hire_date\" RANGE BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW) AS \"$0\""
+        + "\nFROM \"foodmart\".\"employee\"";
```

Comment:
`RANGE BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW` should be the default and not print out.

Reply:
Sorry don't get your opinion @zabetak , if you just run the test without my modification, it'll fail, the distinct is lost, more details can be seen in JIRS

Reply:
@danny0405 Hi Danny, seems rel to SQL will print these, if I remove them, the test failed, maybe we can open a new JIRA to not print out these in RelToSql.

Reply:
We can fix it in separate issue and PR.

Reply:
ok